feature-libs/checkout/b2b/components/checkout-review-submit/checkout-review-submit.component.html

```diff
@@ -184,10 +184,7 @@ <h2 class="cx-review-title d-none d-lg-block d-xl-block">
     <div class="cx-review-cart-heading d-block d-lg-none d-xl-none">
       {{ 'checkoutReview.placeOrder' | cxTranslate }}
     </div>
-    <div
-      class="cx-review-cart-item col-md-12"
-      *ngIf="entries$ | async as entries"
-    >
+    <div class="cx-review-cart-item" *ngIf="entries$ | async as entries">
```

Comment:
Likely a change that should show in the migration docs.

Reply:
Added